_data-prepper/pipelines/pipelines.md

```diff
@@ -84,269 +88,8 @@ conditional-routing-sample-pipeline:
-          log: [ "%{COMMONAPACHELOG}" ]
-  sink:
-    - opensearch:
-        hosts: [ "https://localhost:9200" ]
-        # Change to your credentials
-        username: "admin"
-        password: "admin"
-        # Add a certificate file if you are accessing an OpenSearch cluster with a self-signed certificate  
-        #cert: /path/to/cert
-        # If you are connecting to an Amazon OpenSearch Service domain without
-        # Fine-Grained Access Control, enable these settings. Comment out the
-        # username and password above.
-        #aws_sigv4: true
-        #aws_region: us-east-1
-        # Since we are Grok matching for Apache logs, it makes sense to send them to an OpenSearch index named apache_logs.
-        # You should change this to correspond with how your OpenSearch indexes are set up.
-        index: apache_logs
-```
-
-This example uses weak security. We strongly recommend securing all plugins which open external ports in production environments.
-{: .note}
-
-### Trace analytics pipeline
-
-The following example demonstrates how to build a pipeline that supports the [Trace Analytics OpenSearch Dashboards plugin]({{site.url}}{{site.baseurl}}/observability-plugin/trace/ta-dashboards/). This pipeline takes data from the OpenTelemetry Collector and uses two other pipelines as sinks. These two separate pipelines index trace and the service map documents for the dashboard plugin.
-
-Starting from Data Prepper 2.0, Data Prepper no longer supports `otel_trace_raw_prepper` processor due to the Data Prepper internal data model evolution. 
-Instead, users should use `otel_trace_raw`.
-
-```yml
-entry-pipeline:
-  delay: "100"
-  source:
-    otel_trace_source:
-      ssl: false
-  buffer:
-    bounded_blocking:
-      buffer_size: 10240
-      batch_size: 160
-  sink:
-    - pipeline:
-        name: "raw-pipeline"
-    - pipeline:
-        name: "service-map-pipeline"
-raw-pipeline:
-  source:
-    pipeline:
-      name: "entry-pipeline"
-  buffer:
-    bounded_blocking:
-      buffer_size: 10240
-      batch_size: 160
-  processor:
-    - otel_trace_raw:
-  sink:
-    - opensearch:
-        hosts: ["https://localhost:9200"]
-        insecure: true
-        username: admin
-        password: admin
-        index_type: trace-analytics-raw
-service-map-pipeline:
-  delay: "100"
-  source:
-    pipeline:
-      name: "entry-pipeline"
-  buffer:
-    bounded_blocking:
-      buffer_size: 10240
-      batch_size: 160
-  processor:
-    - service_map_stateful:
-  sink:
-    - opensearch:
-        hosts: ["https://localhost:9200"]
-        insecure: true
-        username: admin
-        password: admin
-        index_type: trace-analytics-service-map
-```
-
-To maintain similar ingestion throughput and latency, scale the `buffer_size` and `batch_size` by the estimated maximum batch size in the client request payload.
-{: .tip}
-
-### Metrics pipeline
```

Comment:
Deleted this example because it's redundant to https://github.com/opensearch-project/documentation-website/pull/6248 (waiting for approval to merge PR and publish)